.github/workflows/labeler-theme.yml

```
@@ -0,0 +1,38 @@
+# Default labeler rules for WordPress block/theme repositories
+#
+# Assigns labels based on file paths to signal where changes occur within block
+# theme projects. These patterns assume a fairly standard theme structure.
+
+area:
```

actionlint (1.7.7)

unexpected key "area" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)